mix.exs

```diff
@@ -589,6 +591,10 @@ defmodule EMQXUmbrella.MixProject do
     ] ++ build_info()
   end
 
+  defp default_cookie() do
+    "emqx50elixir"
```

Comment:
Is this to prevent an Elixir release node from connecting to an Erlang one?

Reply:
yes.
it's technically possible, but we don't want to encourage so before we test it.